app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,80 @@
+    <string name="launch_game">Launch</string>
+    <string name="join_game_text">Type the ID of the game you want to join !</string>
+    <string name="join_id_hint">000</string>
+    <string name="join_button_text">Join</string>
+    <string name="join_fail_format">Failed to join lobby %d.</string>
+    <string name="lobby_not_found">Lobby %d not found.</string>
+    <string name="current_round_format">Current round: %d</string>
+    <string name="singing_format">%s is singing!</string>
+    <string name="guess_the_song">Guess the song!</string>
+    <string name="open_game_error">Failed to open the game.</string>
+    <string name="launch_game_error">Failed to launch the game.</string>
+    <string name="add_a_playlist_manually">Add a playlist manually</string>
+    <string name="song_name">Song Name</string>
+    <string name="artist_name">Artist Name</string>
+    <string name="create_playlist">CREATE PLAYLIST</string>
+    <string name="add_song">ADD SONG</string>
+    <string name="lyricsPresenter">lyrics :</string>
+    <string name="pick_a_song">Pick a song!</string>
+    <string name="submit">Submit</string>
+    <string name="playlist_name">Playlist Name</string>
+    <string name="artistLyricsDisplay">Artist :</string>
+    <string name="songLyricsDisplay">Song :</string>
+    <string name="welcome_to_google_signin">Welcome to Google SignIn</string>
+    <string name="your_spotify_playlists_are_loading">Your Spotify playlists are loading...!</string>
+    <string name="this_could_take_some_time">This could take some time...!</string>
+    <string name="you_have_successfully_logged_into_spotify">You have successfully logged into Spotify!</string>
+    <string name="import_my_spotify_playlists">Import my Spotify playlists</string>
+    <string name="deletePlaylist">Delete</string>
+    <string name="editPlaylist">Edit</string>
+    <string name="sign_in_message">Sign in</string>
+    <string name="sign_out_message">Sign out</string>
+    <string name="click_on_a_song_to_delete_it">Click on a song to delete it.</string>
+    <string name="wait_for_pick_format">Waiting on %s to pick a song</string>
+    <string name="congrats">Congratulations!</string>
+    <string name="micPermissionError">The microphone permissions could not be updated.</string>
+    <string name="pref_locale">fa</string>
```

Comment:
What does fa stand for?

Reply:
This is a relic of an earlier version of language change. I forgot to remove it, thanks for the comment.